handlers/handlers.go

```diff
@@ -15,6 +17,22 @@ func Init(sw wrapper.SlackWrapper) {
 	slackWrapper = sw
 }
 
+// HelpCallback is a handler that takes a dialogCallback, generated by the HelpRequest
+// handler and logs the help request
+func HelpCallback(res *server.Response, req *server.Request, ctx interface{}) error {
+	s, ok := ctx.(string)
+	if !ok {
+		return fmt.Errorf("Expected a string to be passed to the handler")
+	}
+	var d *slack.DialogCallback
```

Comment:
I am not sure you need to create a pointer here. Basically you have declared that d is a pointer to a slack.DialogCallback and then on the next line asked for the address of d. That is now a pointer to a pointer. tl;dr delete the asterisk or the ampersand.